pkg/server/server.go

```diff
@@ -863,8 +863,12 @@ func (s *ImmuServer) CreateDatabaseV2(ctx context.Context, req *schema.CreateDat
 	s.logDBOptions(db.GetName(), dbOpts)
 
 	err = s.startReplicationFor(db, dbOpts)
-	if err != nil && err != ErrReplicatorNotNeeded {
-		return nil, fmt.Errorf("%w: while starting replication", err)
+	if err != nil {
+		if err == ErrReplicatorNotNeeded {
```

Comment:
it's better to compare errors using `errors.Is()` because it will cover the case the error is wrapped